data/query.go

```diff
@@ -0,0 +1,97 @@
+package data
+
+import (
+	"context"
+	"net/url"
+	"strconv"
+
+	"github.com/ONSdigital/log.go/log"
+)
+
+const (
+	// DefaultLimit - default values for limit query
+	DefaultLimit = 10
+	// DefaultSort - default values for sort query
+	DefaultSort = "relevance"
+	// DefaultPage - default values for page query
+	DefaultPage = 1
+)
+
+// GetLimitOptions returns all available limit options for search
+func GetLimitOptions() []int {
+	return []int{10, 25, 50}
+}
+
+// UpdateQueryWithOffset - removes page key and adds offset key to query to be then passed to dp-search-query
+func updateQueryWithOffset(ctx context.Context, query url.Values) url.Values {
+
+	page, err := strconv.Atoi(query.Get("page"))
+	if err != nil {
+		log.Event(ctx, "unable to convert search page to int - set to default "+strconv.Itoa(DefaultPage), log.INFO)
```

Comment:
This works but the Go Idiomatic way to do this would be to use the `fmt.Sprintf()` function:
```
log.Event(ctx, fmt.Sprintf("unable to convert search page to int - set to default %d", DefaultPage), log.INFO)
```
It's a bit easier to read (IMO).

Reply:
I was wondering if we should even be setting a default value the input is incorrect? Is this the current standard for our controllers?

Reply:
I couldn't find documentation on standards for the controller. But on the current ONS search page, if the user changes the `size` parameter (which is `limit` in our case) to a value less than 10, it will automatically set the limit to 10. This change is achieved by the user changing the `size` value from the URL itself. Otherwise, the user is only limited to changing the value through the dropdown select on the bottom of the search page itself. 

When it comes to the `page` numbers, you will get a Page not Found (404) if the user asks for a page outside the available pages. This change is also similar to the previous